src/createStore/createStore.js

```diff
@@ -11,11 +11,21 @@ const defaultOptions = {
   listeners: undefined,
   devtools: true,
   name: 'store',
+  drivers: {
+    http: (...args) => fetch(...args),
+  },
 }
 
 export default (definition, options = defaultOptions) => {
```

Comment:
`options = defaultOptions` is useless

Reply:
This is not uselesss since I use options.drivers